cg/meta/store/base.py

```diff
@@ -100,3 +81,18 @@ def get_tags(file: dict, pipeline_tags: list) -> list:
     sorted_tags = sorted(list(only_existing_tags))
 
     return sorted_tags
+
+
+def build_bundle(config_data: dict, sampleinfo_data: dict, deliverables: dict) -> dict:
```

Comment:
```suggestion
def build_bundle(config_data: dict, analysisinfo_data: dict, deliverables: dict) -> dict:
```
sampleinfo_data feels very MIP specific and this is in the base.py